devops/docker/ubuntu-14.04-amd64/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:trusty
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt -y update && apt-get -y install software-properties-common
+RUN apt -y update && apt-get -y install \
+    apt-transport-https \
+    git \
+    cmake \
+    build-essential \
+    curl \
+    libcurl4-openssl-dev \
+    libssl-dev \
+    uuid-dev \
+    ninja-build \
+    python3-jsonschema \
+    wget \
+    gcovr\
+    jq \
+    bc \
+    file
```

Comment:
We should be able in the future to reduce this list.

Reply:
I used the same packages from other distros with the exception of CMake, rapidjson, GTest. Although not all are required for building (eg gcovr, bc) other utilities are leveraged by the other workflows for static analysis.